julian-mainardis-globant/marvel-app
# Dialog Fragment with details of an especific character added

## Description

Added a Dialog Fragment showing the details of the character selected

## Story Type

- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?

Playing the emulator and pressing the button in the main activity, then you have to scroll in the middle of the screen to see all the characters, after that, you have to click in one of them and a fragment will appear.

## Evidences

| First Header  | Second Header |
| ------------- | ------------- |
| ![pixel3](https://user-images.githubusercontent.com/109156192/191811378-8bc43d59-c535-4838-b4ff-c9865ec84fc8.gif) | ![nexusS](https://user-images.githubusercontent.com/109156192/191811412-a4054d1c-b384-47a9-a909-ecd1a29821d3.gif) |

## Coverage
<img width="450" alt="image" src="https://user-images.githubusercontent.com/109156192/191997548-57632de4-6acc-434e-aaef-05cba9b18da4.png">
<img width="450" alt="image" src="https://user-images.githubusercontent.com/109156192/191997614-9bdfeb96-c8c4-4ecd-ae10-f012805828f1.png">
<img width="450" alt="image" src="https://user-images.githubusercontent.com/109156192/191997771-b4a6eea9-86cd-4c5d-b6f8-6aafbdb69e8b.png">

## Checklist

- All the unit test passed :white_check_mark:

## Tested On
- Pixel 3
- Nexus S

## Package Structure
<img width="254" alt="image" src="https://user-images.githubusercontent.com/109156192/191811193-f4ca1b8a-7d5e-4f42-9fc9-2dfdbe10dcbd.png"> <img width="257" alt="image" src="https://user-images.githubusercontent.com/109156192/191811287-4b2c8d9a-eae4-4034-a1c8-934aa496dd69.png">
